builtin/credential/github/backend.go

```diff
@@ -43,6 +43,20 @@ func Backend() *backend {
 		OperationPrefix: operationPrefixGithub,
 		OperationSuffix: "team-mapping",
 	}
+	teamMapPaths[0].Operations = map[logical.Operation]framework.OperationHandler{
+		logical.ListOperation: &framework.PathOperation{
+			Callback: teamMapPaths[0].Callbacks[logical.ListOperation],
+			Summary:  teamMapPaths[0].HelpSynopsis,
+		},
+		logical.ReadOperation: &framework.PathOperation{
+			Callback: teamMapPaths[0].Callbacks[logical.ReadOperation],
+			Summary:  teamMapPaths[0].HelpSynopsis,
+			DisplayAttrs: &framework.DisplayAttributes{
+				OperationSuffix: "teams2", // The ReadOperation is redundant with the ListOperation
```

Comment:
```suggestion
				OperationVerb:   "list",
				OperationSuffix: "teams2", // The ReadOperation is redundant with the ListOperation
```

This will result in `github-list-teams2` instead of `github-read-teams2`